Update/16128 add attribute types documentation (#16220)
WordPress/gutenberg: docs/designers-developers/developers/block-api/block-attributes.md

}
// { "url": "https://lorempixel.com/1200/800/" }
```

> #### `attribute` Type Validation

Reviewer: ```suggestion

#### `attribute` Type Validation
```

Accepted values in the `type` field of an `attribute` MUST be one of the following:

* null
* boolean
* object
* array
* number
* string
* integer
	- Matches any number with a zero fractional part

> Additional details are found in the documentation for Wordpress's REST API [here](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/).

Reviewer 2: Here's a suggestion for improving the link:

```suggestion
See [Wordpress's REST API documentation](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/) for additional details.
```

Reviewer: Also, capital P :)

```suggestion
See [WordPress's REST API documentation](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/) for additional details.
```

Reviewer 2: hah, how'd I miss that? :-)
